diagnostic.md

```diff
@@ -32,14 +38,16 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+getter method > css() (both getter / setter), val()
+Setter method > Append(), prepend()
```

Comment:
Also remember that getter methods in jQuery become setters when you pass them an argument to use as the value.